app/theme.js

```diff
@@ -1,10 +1,59 @@
 "use client";
-import { createTheme } from "@mui/material/styles";
+import { createTheme } from "@mui/material";
+import Poppins from "@fontsource/poppins";
+import tokens from "./tokens.json";
 import Montserrat from "typeface-montserrat";
 
 const theme = createTheme({
+  palette: {
+    primary: {
+      main: tokens.global.md.sys.color["enabled-button-color"].value,
+    },
+    secondary: {
+      main: tokens.global.md.sys.color["pressed-button-color"].value,
+    },
+    background: {
+      main: tokens.global.md.sys.color["main-background-color"].value,
+      card: tokens.global.md.sys.color["card-background-color"].value,
+    },
+    button: {
+      pressed: tokens.global.md.sys.color["pressed-button-color"].value,
+      enabled: tokens.global.md.sys.color["enabled-button-color"].value,
+    },
+    card: {
+      background: "#FAF5F3",
+      shadow:
+        "0px 1px 3px 1px rgba(0, 0, 0, 0.15), 0px 1px 2px 0px rgba(0, 0, 0, 0.30)",
+    },
+    progressBar: {
+      filled: tokens.global.md.sys.color["filled-progress-bar-color"].value,
+      unfilled: tokens.global.md.sys.color["unfilled-progress-bar-color"].value,
+    },
+  },
   typography: {
-    fontFamily: "Montserrat, sans-serif",
+    fontFamily: "Poppins, sans-serif",
```

Comment:
We'll have to figure out the capitalization problem because right now the button text symbol for flat is 'B'